ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -136,6 +136,13 @@ ZTC:
   opt_uses:
   - zone_air_deadband_temperature_setpoint
   - zone_air_relative_humidity_sensor
+  - zone_air_relative_humidity_status
```

Comment:
not valid. relative humidity is a %; what does a relative humidity status even indicate? that there is humidity?